oar/cli/cmd_update_bug_list.py

```diff
@@ -37,6 +40,9 @@ def update_bug_list(ctx, notify, confirm_droppable):
             if confirm_droppable:
                 high_severity_issues, _ = JiraManager(cs).get_high_severity_and_can_drop_issues(jira_issues)
                 NotificationManager(cs).share_high_severity_bugs(high_severity_issues)
+            elif notify_managers:
+                unverified_cve_issues = JiraManager(cs).get_unverified_cve_issues(jira_issues)
+                NotificationManager(cs).share_unverified_cve_issues_to_managers(unverified_cve_issues)
```

Comment:
suggest to check if issue list is empty

Reply:
This is covered here https://github.com/openshift/release-tests/pull/541/files#diff-a381d0d7c9b68a8747eb48b4124cdba8d709b5da46fb0b26d9e539a45b729e6cR544

Reply:
IMO, even notification api can handle this case, in this case to make the logic more clear and meaningful, we should not call notification func if there is no unverified cve issue found. implicit handling is not a good practice.

Reply:
I added the check for `unverified_cve_issues` and also for `high_severity_issues`. Notification methods won't be called if the lists are empty.